modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Runtime/LegacyCliOptions.cs

```diff
@@ -144,6 +144,8 @@ public class LegacyCliOptions : Dictionary<string, string>, IAgentConfigProvider
                         (int k) => this[LegacyCliConfigKeys.BatchTriggerInterval] = TimeSpan.FromSeconds(k).ToString() },
                     { "ms|iothubmessagesize=", "The maximum size of the (IoT D2C) message.",
                         (int i) => this[LegacyCliConfigKeys.MaxMessageSize] = i.ToString() },
+                    { "mom|maxoutgressmessages=", "The maximum amount of the (IoT D2C) messages",
```

Comment:
Maybe just `om` to keep argument naming consistent ?

Also, please mention that this is ougress buffer size. With just `The maximum amount of the (IoT D2C) messages` I am left wondering whether publisher will stop sending messages after this number or what.

Reply:
changed to om and updated description